docs/tiltfile_config.md

```diff
@@ -102,7 +104,7 @@ if 'd' in to_edit:
   docker_build('d', './d')
 ```
 
-## The Config File
+## How Config Works
```

Comment:
this section is actually specifically about the file and not generally about how configuration works